app/models/vote.rb

```diff
@@ -0,0 +1,4 @@
+class Vote < ApplicationRecord
```

Comment:
You should also include a validation to ensure the combination of user_id and work_id  are unique.  
```suggestion
class Vote < ApplicationRecord
  validates :work_id, uniqueness: { scope: :user_id,
    message: "You can only vote for a work once" }
```